tests/framework/lookup/test_lookup.py

```diff
@@ -195,7 +202,13 @@ def test_invalid_data_type_build_table(base_config):
 def test_lookup_table_interpolated_return_types(base_config):
     year_start = base_config.time.start.year
     year_end = base_config.time.end.year
-    data = build_table(lambda age, sex, year: year, year_start, year_end)
+    data = build_table(
+        lambda x: x[0],
+        parameter_columns={
```

Comment:
I think there should be a test that includes 1 parameter col and more than 2 parameter cols since it's been generalized